### PR TITLE
return empty when the TagName OR edgeName of current row is different…

### PR DESCRIPTION
**Pull request by nevermore3**
… from the TagName OR edgeName to be processed

### CODE REVIEW

File: src/storage/context/StorageExpressionContext.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ Value StorageExpressionContext::getSrcProp(const std::string& tagName,
                                            const std::string& prop) const {
     if (!isEdge_ && reader_ != nullptr) {
         if (tagName != name_) {
-            return Value::kNullValue;
+            return Value::kEmpty;
```
Comment (critical27) [inline src/storage/context/StorageExpressionContext.cpp:95]: Line 114 perhaps need to change it as well.

**Review by critical27 — COMMENTED** [review]
What is the conclusion about `NULL` and `EMPTY`?

**Comment by nevermore3** [comment]
> What is the conclusion about `NULL` and `EMPTY`?

From the perspective of the user:  
```
Return EMPTY when the value does not exist
Return NULL when the value is NULL
```

From the perspective of the developer:
```
Return EMPTY when the value does not exist
Return NULL when the value is NULL OR EXCEPTION
```

Try to make the smallest changes to the semantics，Before a Release version。

File: src/storage/context/StorageExpressionContext.cpp (hunk 1 above)
Comment (nevermore3) [reply src/storage/context/StorageExpressionContext.cpp:95]: done

**Comment by czpmango** [comment]
> > What is the conclusion about `NULL` and `EMPTY`?
> 
> From the perspective of the user:
> 
> ```
> Return EMPTY when the value does not exist
> Return NULL when the value is NULL
> ```
> 
> From the perspective of the developer:
> 
> ```
> Return EMPTY when the value does not exist
> Return NULL when the value is NULL OR EXCEPTION
> ```
> 
> Try to make the smallest changes to the semantics，Before a Release version。

What does "EXCEPTION" mean? Won't "EXCEPTION" report an error？

**Comment by nevermore3** [comment]
> > > What is the conclusion about `NULL` and `EMPTY`?
> > 
> > 
> > From the perspective of the user:
> > ```
> > Return EMPTY when the value does not exist
> > Return NULL when the value is NULL
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > From the perspective of the developer:
> > ```
> > Return EMPTY when the value does not exist
> > Return NULL when the value is NULL OR EXCEPTION
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Try to make the smallest changes to the semantics，Before a Release version。
> 
> What does "EXCEPTION" mean? Won't "EXCEPTION" report an error？

during the execution phase, the data is abnormal, 
For example, vid type is different from the schema, return value::kNullBadType

**Review by critical27 — APPROVED** [review]
Good job

**Comment by critical27** [comment]
> > > > What is the conclusion about `NULL` and `EMPTY`?

OK, the rules LGTM

**Review by yixinglu — APPROVED** [review]
We maybe need not the `EMPTY` later, now LGTM.

**Pull request merged**